tasks/hardening.yml

```diff
@@ -18,7 +18,7 @@
     - _users_status == "unset" or
       _users_status == "set_bash"
 
-- name: "hardening | ensure ~{{ item.key }}/.bash_history is immutable"
+- name: "hardening | ensure ~{{ item.key }}/.bash_history content cannot be tampered with"
```

Comment:
```suggestion
- name: "hardening | ensure ~{{ item.key }}/.bash_history is in append only mode so that content cannot be tampered with"
```

Reply:
This is a good task name, can you commit/squash it?